audit/auditlog.go

```diff
@@ -33,7 +34,7 @@ func logPackage(noColor bool, idx int, packageCount int, coordinate types.Coordi
 	}
 }
 
-func logVulnerablePackage(noColor bool, idx int, packageCount int, coordinate types.Coordinate) {
+func logVulnerablePackage(noColor bool, idx int, packageCount int, coordinate types.Coordinate) (vulnerableCount bool) {
```

Comment:
`vulnerableCount` is a bool?
Also it doesn't look like the return value is used?

Reply:
Thanks for nixing this in: cdaffbe